### PR TITLE
Resources!

### PR DESCRIPTION
**Pull request by b-n**
The start of a resources section on our site.

Some moots/notes:

- `resources` or `articles`? (The latter is used by our ber folks)
- I assume we want to use the `/admin` interface too (It works! see #40)
- It's currently "disabled" in that there is no reference link in the Menu, however commenting back in the header_themes value for resources makes it available. The links are there and ready to use, but you just need to hack them into the URL manually for now until we're ready to expose it all etc.
- IMO, If there is no translation, we should just remove the if/else condition and just keep it in one language. We could eventually look at an include for "Help us by commiting changes/translation in our github" or something.

:warning: There is a mild problem with permalinks. If `permalink_nl` isn't specified, it will try to render into `/nl/resources/` (which in theory should never exist. The selection in the NL bronnen page will likely still work, however the language selector won't.

## Tests

From the git repo:

1. Copy the template to a file without the prefixed "_"

```bash
cp _resources/_template.md _resources/testing.md
```

2. Edit some of the content - Be amazed at how it "just works"
3. Check it out in the dockers/jekyll directly

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -1 +1 @@
-3.3.0
+3.1.1
```
Comment (b-n) [inline .ruby-version:1]: Downgrading to the same version that is running in the docker compose container.

File: Gemfile.lock [hunk 2]
```diff
@@ -158,4 +158,4 @@ RUBY VERSION
    ruby 3.1.1p18
 
 BUNDLED WITH
-   2.5.4
+   2.3.25
```
Comment (b-n) [inline Gemfile.lock:161]: Likewise

File: _i18n/nl.yml [hunk 3]
```diff
@@ -5,6 +5,7 @@ langs:
 pages:
   about: Over
   events: Agenda
+  resources: Middelen
```
Comment (b-n) [inline _i18n/nl.yml:8]: `Middelen` or `Bronnen` :shrug:

File: _includes/resources.html [hunk 4]
```diff
@@ -1,14 +1,14 @@
-{% unless page.permalink == '/blog' %}
-  <a href="/blog">{% t home.blog.more %}</a>
+{% unless page.permalink == '/resources' %}
```
Comment (b-n) [inline _includes/resources.html:1]: I think we can remove the unless part and just always include the `more` link. If we did have permalinks (which we don't for resources per the frontmatter) I think we should still include the "more" yeah?

File: _includes/resources.html [hunk 5]
```diff
@@ -1,14 +1,14 @@
...
+  <a href="/resources">{% t home.resources.more %}</a>
 {% endunless %}
 <ul
   class="list -no-list-style l-stack -vertical"
   style="--stack-spacing: 1.5rem"
 >
   {% assign limit = include.limit %}
-  {% assign sorted_posts = site.blog | filter_tags: include.tags | sort: 'date' | reverse %}
+  {% assign sorted_posts = site.resources | filter_tags: include.tags | sort: 'date' | reverse %}
```
Comment (b-n) [inline _includes/resources.html:9]: Sorted by date, thoughts about sorting by name instead?

File: _resources/first.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
+---
```
Comment (b-n) [inline _resources/first.md:1]: We should kill this resource, but nice to have a testing resource for now.

File: admin/config.yml [hunk 7]
```diff
@@ -51,3 +51,17 @@ collections:
           widget: 'image',
           required: false,
         }
+  - name: 'resource' # Used in routes, e.g., /admin/collections/resource
```
Comment (b-n) [inline admin/config.yml:54]: This enables editing via `/admin` on our site.

File: _i18n/nl.yml (hunk 3 above)
Comment (csmeyns) [reply _i18n/nl.yml:8]: Personally I'd suggest 'Bronnen' in this case,  I've used it elsewhere as a translation of 'resources' too 👌

File: _includes/resources.html (hunk 4 above)
Comment (csmeyns) [reply _includes/resources.html:1]: Yes agree (I'm not sure what the original motivation was not to show "more" in that case, but if need be we can always add it back)

**Comment by b-n** [comment]
This is almost 100% working.

I think I might have just discovered a way to do localised slugs :sweat_smile:

**Review by csmeyns — APPROVED** [review]
Yes, this works well now 🙂

**Pull request merged**